vite.config.ts

```diff
@@ -64,8 +64,8 @@ export default defineConfig(({command, mode}: ConfigEnv): UserConfig => {
         display: 'swap',
         injectTo: 'head-prepend',
         families: [
-          'Inter:100,200,300,400,500,600,700,800,900',
-          'Lexend:100,200,300,400,500,600,700,800,900'
+          'Inter:wght@100,200,300,400,500,600,700,800,900',
+          'Lexend:wght@100,200,300,400,500,600,700,800,900'
```

Comment:
```suggestion
          // families can be either strings (only regular 400 will be loaded)
          'Lexend',

          // or objects
          {
            /**
             * Family name (required)
             */
            name: 'Inter',

            /**
             * Family styles
             */
            styles: 'ital,wght@0,400;1,200',

            /**
             * enable non-blocking renderer
             *   <link rel="preload" href="xxx" as="style" onload="this.rel='stylesheet'">
             * default: true
             */
            defer: true,
          },
```

So after looking at the [project README](https://github.com/stafyniaksacha/vite-plugin-fonts#options), this will work. It loads only the 400 or regular version of Lexend, and multiple versions of Inter.